src/codegen.cpp

```diff
@@ -5640,6 +5625,25 @@ static void init_julia_llvm_env(Module *m)
                                      "jl_float_temp"));
 #endif
 
+#ifndef JULIA_ENABLE_THREADING
+    jltls_states_var =
+        new GlobalVariable(*m, ArrayType::get(T_int8, sizeof(jl_tls_states_t)),
```

Comment:
I actually had that at first but was a little bit paranoid about the size I should supply since `jl_tls_states_t` has a `int16_t` and is not exactly multiple of pointer size.